src/OrchardCore.Modules/OrchardCore.ContentFields/Assets/trumbowyg.media.tag.js

```diff
@@ -17,17 +17,23 @@
                             $("#mediaApp").show();
                             mediaApp.selectedMedias = [];
                             var modal = $('#mediaModalHtmlField').modal();
+                            //disable an reset on click event over the button to avoid issue if press button multiple times or have multiple editor
+                            $('#mediaHtmlFieldSelectButton').off('click');
                             $('#mediaHtmlFieldSelectButton').on('click', function (v) {
-                                var mediaBodyContent = "";
-                                for (i = 0; i < mediaApp.selectedMedias.length; i++) {
-                                    mediaBodyContent += ' [image]' + mediaApp.selectedMedias[i].mediaPath + '[/image]';
-                                }
-                                var node = document.createTextNode(mediaBodyContent);
                                 trumbowyg.restoreRange();
                                 trumbowyg.range.deleteContents();
-                                trumbowyg.range.insertNode(node);
-                                trumbowyg.syncTextarea();
-                                $(document).trigger('contentpreview:render');
+                                
+                                for (i = 0; i < mediaApp.selectedMedias.length; i++) {
+                                    var mediaBodyContent = ' [image]' + mediaApp.selectedMedias[i].mediaPath + '[/image]';
```

Comment:
Just a little comment about image shortcode, I'm not sure if there's a better way to inject the shortcode instead of writing them directly

/cc @deanmarcussen

Reply:
Seems good to me. This is in javascript, I don't see a better way.

Reply:
The only problem is whenever the actual markup changed for a particular code, no one will knows, for now it's fine, unless Dean has some suggestion

Reply:
No, it's totally fine with me how it is. Hopefully we won't change that again, but if we did, no one will forget to test it out right :)

Reply:
@agriffard so you can merge